### PR TITLE
Implement proper CLI arg parsing for perf.

### PR DESCRIPTION
**Pull request by whiten**
Use this to support various browsers and output styles. All defaults
should be exactly as previously.

Example commands:

% npm run perf
% npm run perf -- --help
% npm run perf -- --devtools
% npm run perf -- --groups lock
% npm run perf -- --browsers firefox webkit
% npm run perf -- --browsers all
% npm run perf -- --format replicache

### CODE REVIEW

File: perf/lock.ts [hunk 1]
```diff
@@ -67,6 +68,9 @@ function benchmarkHeldRead(mode: Mode) {
     group: 'lock',
     id: '',
     tab: null as Tab | null,
+    async skip() {
```
Comment (arv) [inline perf/lock.ts]: doesn't look like you need `async` here

File: perf/runner.js [hunk 2]
```diff
@@ -8,13 +10,101 @@ import * as os from 'os';
...
+    err.value = arg;
+    err.optionName = '--browsers';
+    throw err;
+  }
+  return arg;
+}
+
+function browserName(browser) {
+  let name = browser[0].toUpperCase() + browser.substr(1);
+  if (name === 'webkit') {
```
Comment (arv) [inline perf/runner.js]: never true

File: src/test-util.ts [hunk 3]
```diff
@@ -59,6 +59,13 @@ export function defineTestFunctions(self: Window & typeof globalThis): void {
   }
 }
 
+export function isFirefox(): boolean {
+  return true;
```
Comment (arv) [inline src/test-util.ts]: please fix

**Review by arv — APPROVED** [review]
with nits

File: src/test-util.ts (hunk 3 above)
Comment (whiten) [reply src/test-util.ts]: Yikes, thank you.

File: perf/runner.js (hunk 2 above)
Comment (whiten) [reply perf/runner.js]: Fixed.

File: perf/lock.ts (hunk 1 above)
Comment (whiten) [reply perf/lock.ts]: Removed.

Comment (whiten) [reply perf/lock.ts]: Removed, thank you.

**Pull request merged**